smart_alert.js

```diff
@@ -17,6 +17,14 @@
  * 3. btm-left 画面左下に表示します。
  * 4. btm-right 画面右下に表示します。
  */
+
+let alertsObj = {
```

Comment:
I would prefer to have this as a static property of the SmartAlert class, because I want it to be completed only within the scope of the class.